vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -245,7 +245,7 @@
 
 - name: Ensure ArkCase required services are populated inside the arkcase.service
   set_fact:
-    arkcase_required_services: "{{ (db_engine == 'postgresql') | ternary('postgresql-13.service', 'mariadb.service') }} {{'activemq.service alfresco.service samba.service solr.service config-server.service' if ldap_type=='samba' else 'activemq.service alfresco.service solr.service config-server.service' }}"
+    arkcase_required_services: "{{ (db_engine == 'postgresql') | ternary('postgresql-13.service alfresco7.service', 'mariadb.service alfresco.service') }} {{'activemq.service samba.service solr.service config-server.service' if ldap_type=='samba' else 'activemq.service solr.service config-server.service' }}"
```

Comment:
we need a better solution here, to support multi-VM installs where every service is not on the same VM.  Let's try to think this through.